catalog/core/catalog-core-migratable/src/test/java/org/codice/ddf/catalog/migratable/impl/MigrationFileWriterTest.java

```diff
@@ -210,6 +180,17 @@ public void testWriteMetacardsWriteObjectFails() throws Exception {
         List<Result> results = new ArrayList<>();
         results.add(new ResultImpl(new MetacardImpl()));
 
+        mockFile = new File("");
```

Comment:
Yes but now you have a file left over after your test since it wasn't deleted. If you do want to keep the file, I recommend using the TemporaryFolder or something else to do the job so it is guaranteed to be deleted at the end of the test http://howtodoinjava.com/junit/junit-creating-temporary-filefolder-using-temporaryfolder-rule/. But since you have a mock object already, seems easier just to continue mocking what you need.